lib/inputstreamhelper.py

```diff
@@ -181,12 +181,8 @@ def _cmd_exists(cmd):
 
     def _helper_disabled(self):
         """Return if inputstreamhelper has been disabled in settings.xml."""
-        disabled = ADDON.getSetting('disabled')
-        if not disabled:
-            ADDON.setSetting('disabled', 'false')  # create default entry
-            disabled = 'false'
-
-        if disabled == 'true':
+        disabled = ADDON.getSettingBool('disabled')
```

Comment:
This method has been available since Kodi 18. It's broke backwards compatibility with Kodi 17 and later.

Reply:
I asked whether I should revert this change before.
https://github.com/emilsvennesson/script.module.inputstreamhelper/pull/34#issuecomment-475909581

Reply:
> I'll have to create a new commit changing this back, if that's what you want.
Yes. I meant that. Thanks.

Reply:
Reverted it.